### PR TITLE
Docs: Added Experimental HTTP error patterns

### PR DESCRIPTION
**Pull request by chloepoulterdesign**
I have added a new experimental page for the HTTP errors pattern.

### CODE REVIEW

File: src/pages/experimental/http-errors/index.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+![404 error in context](/images/404.png)
+
+<Caption>Example of a 404 error in context</Caption>
+
+## Consists of
+
+- An illustration (optionally including animation) that can be relevant to the context, the product or be generic
+- The error number
+- Headline text explaining the cause of the error maintaining an informal tone
+
```
Comment (vikkipaterson) [inline src/pages/experimental/http-errors/index.mdx]:
```suggestion
- Optionally consider including description text introducing suggested alternative links to help the user
```

File: src/pages/experimental/http-errors/index.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <AnchorLink>403 Error</AnchorLink>
+  <AnchorLink>404 Error</AnchorLink>
+  <AnchorLink>Other errors</AnchorLink>
+</AnchorLinks>
+
+## 403 Error
+
+Headline text - "Hmm, it appears you don't have access to this resource_name" where resource_name is replaced as needed.
+
+Description text should introduce the suggested page link(s).
```
Comment (vikkipaterson) [inline src/pages/experimental/http-errors/index.mdx]:
```suggestion
Description text should introduce any suggested page link(s).
```

File: src/pages/experimental/http-errors/index.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+![Example of a 403 error page](images/1.png)
+
+ </Column>
+</Row>
+
+## 404 Error
+
+Headline text - "Hmm, we can't seem to find that."
+
+Description text should introduce the suggested page link(s).
```
Comment (vikkipaterson) [inline src/pages/experimental/http-errors/index.mdx]:
```suggestion
Description text should introduce any suggested page link(s).
```

**Comment by connor-leech** [comment]
New experimental content will be accepted via an upcoming submission platform as roughly detailed here: https://app.zenhub.com/workspaces/carbon-design-system-593830641344b813db10934d/issues/carbon-design-system/carbon/4869

I've touched base with @vikkipaterson about working together on shaping that in the new year. Thank you!

**Pull request closed without merging**